R/frgtracker.R

```diff
@@ -1,11 +1,38 @@
-# function1 start
 
 
-# function1 end
+# register_courses start
 
-# function2 start
+#' Read and store the input dataframe as a DataFrame.
+#'
+#' @param df dataframe: A tidy dataframe:
```

Comment:
each argument needs its own "@param". The syntax is also different  - in R we dont start with the type. We usually describe the type in the description itself. please look at some examples : https://kbroman.org/pkg_primer/pages/docs.html